Apps/Quran Revision Companion.md

```
@@ -0,0 +1,24 @@
+# Quran Revision Companion
+
+This is an app for people that are memorizing or have already memorized the holy quran aka huffaz.
+
+## Features
+
+- There are four Mushafs layouts,
+	1. 16 Line Mushaf (IndoPak font)  
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)